utils/hare-bootstrap

```diff
@@ -353,20 +366,22 @@ wait4 ${pids[@]-}
 agents_nr=$(( ${#pids[@]} + 1 ))
 
 # Waiting for the Consul agents to get ready...
-count=1
 while (( $(get_ready_agents_nr) != $agents_nr )); do
-    if (( $count > 5 )); then
-        echo 'Some agent(s) failed to start in due time:' >&2
-        diff <(get_ready_agents | sort) \
-             <(get_all_nodes | awk '{print $1}' | sort) | sed 1d >&2
+    echo 'Consul not ready on node :'
```

Comment:
Please note that this output goes to stdout while consequent `comm` will print the output to stderr.

Doesn't it make sense to redirect output of `echo` to stderr as well?

Reply:
Done, I have redirected both to stdout as they are not really error as yet